tests/test_aws.py

```diff
@@ -154,7 +154,7 @@ def test_vm_subnets(project, latest_amzn_image):
         )
 
     # set both
-    with pytest.raises(ValueError):
+    with pytest.raises(ExternalException):  # NOQA F821
```

Comment:
```suggestion
    with pytest.raises((ExternalException, ValueError)):  # NOQA F821
```
If we just change the exception to `ExternalException` I think the module set tests will start failing for the stable products.

Reply:
I agree. But we should also add a comment saying why the `ValueError` is still there. Otherwise after a few months nobody will know why it's there.

Reply:
I'm unfamiliar with multiple exceptions being raised, I will look into it !